downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -82,12 +82,20 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
 ## Developer tools updates
 
 ### New features
+#### OpenAPI Tool
+Added Swagger version 2 support for OpenAPI to Ballerina client, service generation.
 
 #### Language Server
 
 ### Improvements
 
 #### OpenAPI Tool
+- Added mapping for the `additionalProperty` property in the OpenAPI object schema. With this support, generated Ballerina records can be open records or closed records according to the `additionalProperty` property details.
+  Thereby, some of your existing generated Ballerina records may change into close records after regenerating using Ballerina 2201.4.0 (Swan Lake) version.
+- Improved OpenAPI schema `nullable:true` property support by providing generated record field with default of value nil, instead of making field both nilable and optional.
```

Comment:
How about this explanation with examples since the explanation is bit confusing ? @lnash94 

```suggestion
- Improved OpenAPI schema `nullable:true` property support by providing generated record field with default of value nil (e.g. : `string name = ()` ), instead of making field both nilable and optional (e.g. : `string? name?;`).
```

Reply:
+1, and we can remove the additional Info section. https://github.com/ballerina-platform/ballerina-dev-website/pull/6277/files#r1088560147